pkg/eventhandler/fake/eventhandlers_mock.go

```diff
@@ -180,31 +180,17 @@ func (mr *MockK8sMockRecorder) ConnectToCluster() *gomock.Call {
 }
 
 // CreateK8sJob mocks base method.
-func (m *MockK8s) CreateK8sJob(arg0 string, arg1 *config.Action, arg2 config.Task, arg3 keptn.EventProperties, arg4 k8sutils.JobSettings, arg5 interface{}, arg6 string) error {
+func (m *MockK8s) CreateK8sJob(arg0 string, arg1 *config.Action, arg2 string, arg3 config.Task, arg4 keptn.EventProperties, arg5 k8sutils.JobSettings, arg6 interface{}, arg7 string) error {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "CreateK8sJob", arg0, arg1, arg2, arg3, arg4, arg5, arg6)
+	ret := m.ctrl.Call(m, "CreateK8sJob", arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7)
 	ret0, _ := ret[0].(error)
 	return ret0
 }
 
 // CreateK8sJob indicates an expected call of CreateK8sJob.
-func (mr *MockK8sMockRecorder) CreateK8sJob(arg0, arg1, arg2, arg3, arg4, arg5, arg6 interface{}) *gomock.Call {
+func (mr *MockK8sMockRecorder) CreateK8sJob(arg0, arg1, arg2, arg3, arg4, arg5, arg6, arg7 interface{}) *gomock.Call {
 	mr.mock.ctrl.T.Helper()
-	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "CreateK8sJob", reflect.TypeOf((*MockK8s)(nil).CreateK8sJob), arg0, arg1, arg2, arg3, arg4, arg5, arg6)
-}
-
-// ExistsServiceAccount mocks base method.
```

Comment:
It seems we lost this function from the interface, yet I don't see any related changes in this PR, why is this change here ?

Reply:
I think the function was only present in the mocking code or the implementation of #229. I think it somehow slipped into a previous PR.